port/unix/omrcpu.c

```diff
@@ -63,20 +63,21 @@ omrcpu_startup(struct OMRPortLibrary *portLibrary)
 {
 	/* initialize the ppc level 1 cache line size */
 #if defined(RS6000) || defined (LINUXPPC) || defined (PPC)
-	int32_t ppcCacheLineSize;
-
-	int  i;
+	int32_t ppcCacheLineSize = 0;
+	int32_t i = 0;
 	char buf[1024];
-	memset(buf, 255, 1024);
 
-	__asm__(
-		"dcbz 0, %0"
-		: /* no outputs */
-		:"r"((void *) &buf[512]));
+	memset(buf, 255, sizeof(buf));
+
+	__asm__ __volatile__(
```

Comment:
I suspect the `__volatile__` isn't needed. The port library copy doesn't use it.

Reply:
Ah nm, I see you just added it here. I guess it doesn't hurt.

Reply:
It was used in both places in `j9memclr.cpp` and won't hurt. In fact, it may save us from a future compiler optimization.